app/deck/1.11.x/guides/defaults.md

```diff
@@ -220,40 +230,41 @@ Summary:
 
 3. Save the file and run a diff:
 
-{% capture deck_diff3 %}
+{{ deck_diff2 | indent | replace: " </code>", "</code>" }}
+
+    Notice that the diff doesn't show extra changes.
```

Comment:
Is this missing a step? I don't see the `sync` stage, after which a `diff` would not show extra changes.